DependencyInjection/SonataAdminExtension.php

```diff
@@ -90,6 +90,7 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setParameter('sonata.admin.configuration.dashboard_blocks', $config['dashboard']['blocks']);
         $container->setParameter('sonata.admin.configuration.sort_admins', $config['options']['sort_admins']);
         $container->setParameter('sonata.admin.configuration.breadcrumbs', $config['breadcrumbs']);
+        $container->setParameter('sonata.admin.configuration.role_sonata_admin', $config['security']['roles']['role_sonata_admin']);
```

Comment:
I tend to prefer not using a parameter for 2 reasons:
 - It creates an unintended extension point. People might try to replace this parameter instead of using the config option.
 - The parameter stay in compiled container: it's not really important for one parameter but if you do it to much, it starts cluttering the container for nothing.